src/Service.Host/client/src/components/PurchaseOrders/OverbooksSearch.js

```diff
@@ -16,28 +18,31 @@ import history from '../../history';
 import config from '../../config';
 
 function OverbooksSearch() {
+    const useStyles = makeStyles(theme => ({
+        button: {
+            marginLeft: theme.spacing(1),
+            marginTop: theme.spacing(4)
+        },
+        a: {
+            textDecoration: 'none'
+        }
+    }));
+    const classes = useStyles();
     const dispatch = useDispatch();
     useEffect(() => dispatch(purchaseOrdersActions.fetch()), [dispatch]);
     useEffect(() => dispatch(purchaseOrdersActions.fetchState()), [dispatch]);
 
-    const searchOverbookItems = searchTerm => dispatch(purchaseOrdersActions.search(searchTerm));
-    const searchResults = useSelector(state =>
-        collectionSelectorHelpers.getSearchItems(
-            state.purchaseOrders,
-            100,
-            'orderNumber',
-            'orderNumber',
-            'orderNumber'
-        )
-    );
-
     const item = useSelector(state =>
         collectionSelectorHelpers.getApplicationState(state.purchaseOrders)
     );
     const canSearch = utilities.getHref(item, 'allow-over-book-search');
-    const searchLoading = useSelector(state =>
-        collectionSelectorHelpers.getSearchLoading(state.purchaseOrders)
-    );
+
+    const [options, setOptions] = useState({ orderNumber: '' });
```

Comment:
I would just do const [orderNumber, setOrderNumber] = useState('') as opposed to having an options object (if orderNumber is the only option there's no point in wrapping it in an object)